frontend/www/app/shared/config.js

```diff
@@ -2,7 +2,7 @@ angular.module('civis.youpower')
 
 .factory('Config', function() {
   return {
-     // host: 'http://localhost:3000'
-     host: 'https://app.civisproject.eu'
+     host: 'http://localhost:3005'
```

Comment:
This is test host address. Should not be committed as it will break the production.